gm4_orb_of_ankou/generate_pneumas.py

```diff
@@ -95,15 +95,17 @@ def generate_pneuma_predicates(ctx: Context):
   pneuma: Any = ""
   for pneuma in pneumas:
     # Predicate to check if a player has a certain pneuma equipped
-    nbt = "{gm4_orb_of_ankou:{pneumas:[{id:'"+ pneuma + "'}]}}"
+    custom_data = "{gm4_orb_of_ankou:{pneumas:[{id:'"+ pneuma + "'}]}}"
     ctx.data[f"gm4_orb_of_ankou:pneuma_equipped/{pneuma}"] = Predicate({
       "condition": "minecraft:entity_properties",
       "entity": "this",
       "predicate": {
         "equipment": {
           "offhand": {
-            "tag": "gm4_orb_of_ankou:pneuma_container",
-            "nbt": nbt
+            "items": "#gm4_orb_of_ankou:pneuma_container",
+            "components": {
+              "minecraft:custom_data": custom_data
```

Comment:
This will do an exact component check. Instead use an item sub predicate `"predicates"` instead of `"components"`

Reply:
Thanks, I've updated it to use that